WebHostLib/tracker.py

```diff
@@ -264,16 +264,17 @@ def get_static_room_data(room: Room):
     multidata = Context.decompress(room.seed.multidata)
     # in > 100 players this can take a bit of time and is the main reason for the cache
     locations: Dict[int, Dict[int, Tuple[int, int, int]]] = multidata['locations']
-    names: Dict[int, Dict[int, str]] = multidata["names"]
-    games = {}
+    names: Dict[int, Dict[int, str]] = multidata.get("names", {})
+    games = multidata.get("games", {})
     groups = {}
     custom_locations = {}
     custom_items = {}
     if "slot_info" in multidata:
-        games = {slot: slot_info.game for slot, slot_info in multidata["slot_info"].items()}
-        groups = {slot: slot_info.group_members for slot, slot_info in multidata["slot_info"].items()
+        slot_info_dict: Dict[int, NetworkSlot] = multidata["slot_info"]
+        games = {slot: slot_info.game for slot, slot_info in slot_info_dict.items()}
+        groups = {slot: slot_info.group_members for slot, slot_info in slot_info_dict.items()
                   if slot_info.type == SlotType.group}
-
+        names = {0: {slot: slot_info.name for slot, slot_info in slot_info_dict.items()}}
```

Comment:
This is wrong. `names` was List[List[str]] in multidata, but besides awkward typing, this is off by 1. It's being used as `player_name = names[tracked_team][tracked_player - 1]`